src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -481,6 +489,11 @@ fn make_data(
                     chapter.insert("section".to_owned(), json!(section.to_string()));
                 }
 
+                chapter.insert(
+                    "has_sub_items".to_owned(),
+                    json!((ch.sub_items.len() > 0).to_string()),
```

Comment:
Can probably use is_empty instead of checking the length

Reply:
fixed 686d325